packages/twenty-server/src/engine/integrations/environment/environment-variables.ts

```diff
@@ -231,6 +231,18 @@ export class EnvironmentVariables {
   @IsString()
   SENTRY_DSN: string;
 
+  @ValidateIf(
+    (env) => env.EXCEPTION_HANDLER_DRIVER === ExceptionHandlerDriver.Sentry,
+  )
+  @IsString()
+  SENTRY_RELEASE: string;
```

Comment:
I feel it should be optional even if sentry driver is set. We should not force users to set another environmennt variable